src/migrations/1744398181271_scene-stream-ttl.ts

```diff
@@ -0,0 +1,20 @@
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate'
+
+export const SceneStreamTTLColumns: ColumnDefinitions = {
+  streaming: { type: 'boolean', notNull: true, default: false },
+  start_streaming_at: { type: 'bigint', notNull: false }
+}
+
+export async function up(pgm: MigrationBuilder): Promise<void> {
+  pgm.addColumns('scene_stream_access', SceneStreamTTLColumns)
+  pgm.createIndex('scene_stream_access', ['ingress_id', 'streaming', 'active'], {
```

Comment:
This new index is being built of 3 columns, `ingress_id`, `streaming` and `active`, but the queries describe above are using only `ingress_id` and `active`, so it won't be used.
It's also setting it as unique, will those values be always unique?